feat: add kmeans documentation (#17)
SFTechnologies/spectrify-docs: pages/workspace-management/save-workspace.mdx

import Image from 'next/image';

import WorkspaceManagementFooterLinks from '/components/WorkspaceManagementFooterLinks';

export const imagePath = '/images/workspace-management/';

## Save workspace

<hr></hr>
<br></br>
<Image
  src={`${imagePath}save_workspace_icon.svg`}
  alt='save workspace icon'
  width={65}
  height={65}
/>

Allows users to save the current state of their workspace for future use in **.spws** extension.

> This includes all spectra, analysis and any other relevant settings and results. When the workspace is reopened, it will restore the saved layout and settings exactly as they were when the workspace was saved.

Reviewer: ```suggestion
This includes all spectra, analysis, and other relevant settings and results. When the workspace is reopened, all of this data will be restored so you can continue your work where you left it.
```

Author: Done
